themes/pycon-ca-2018/templates/base.tmpl

```diff
@@ -64,9 +58,7 @@
     </nav>
 
     {% block center_section %}
-        <div class="container">
```

Comment:
I liked having a container wrapper. Without it, everything stretches out, eg = https://deploy-preview-14--peaceful-almeida-642df8.netlify.com/code-of-conduct/ 

you sure about this?

Reply:
Yes. Look now.